core/app/models/spree/address.rb

```diff
@@ -138,5 +145,10 @@ def country_iso=(iso)
     def country_iso
       country && country.iso
     end
+
+    def validate_name
```

Comment:
Can you please explain the reasoning behind this validation logic?

Reply:
@kennyadsl we are going life with the MVP this week, but we can definetly handle this down the road a week from now.